layers/meta-balena-raspberrypi/recipes-core/images/balena-image.bbappend

```diff
@@ -12,8 +12,10 @@ SDIMG_KERNELIMAGE:raspberrypi3-64 ?= "kernel8.img"
 SDIMG_KERNELIMAGE:raspberrypi0-2w-64 ?= "kernel8.img"
 
 # Increase Root File system size
-# ToDo (Waqar): Find correct size and fix issues
-IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "2785280"
+IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "319488"
+IMAGE_OVERHEAD_FACTOR:revpi-connect-s ?= "1.0"
```

Comment:
This typically gives the image enough room to boot and allows for basic post installs while still leaving a small amount of free disk space. (1.0 is the default 30%)